packages/controllers/src/snaps/json-schemas/SnapManifest.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
should we check this in? or have it as a `postinstall`

Reply:
I'm not completely sure how to think about it. It seems harmless to check in? CI will blow up if it changes unexpectedly. I think there is something nice about having it recorded in the git history as well.

Reply:
Yea. I'm not partial to either or.

I know golang culture is totally fine with checked in generated code. In JS culture I've found people like to keep the repo tree clean of generated files.

In Golang the convention is to put a comment at the top to warn people not to edit checked-in generated code:

```
// Code generated by stringer -type Pill pill.go; DO NOT EDIT.
```

Reply:
Done in: e9acf13